settings.gradle

```diff
@@ -38,21 +36,20 @@ ext.isMasterBuild = false ||
     (System.getenv("GITHUB_REF") == "refs/heads/master") ||
     (System.getenv("BUILD_SOURCEBRANCHNAME") == "master")
 
+ext.isCI = System.getenv("CI") != null
+
 buildCache {
     local {
-        enabled = !isMasterBuild
+        enabled = !isCi
     }
-
-    remote(ch.myniva.gradle.caching.s3.AwsS3BuildCache) {
-        endpoint = 'fra1.digitaloceanspaces.com'
-        region = 'fra1'
-        bucket = 'testcontainers'
-        path = 'cache'
-        reducedRedundancy = false
-        push = isMasterBuild && !System.getenv("READ_ONLY_REMOTE_GRADLE_CACHE")
-        headers = [
-            'x-amz-acl': 'public-read'
-        ]
+    remote(HttpBuildCache) { 
+        push = isCI
```

Comment:
Gradle colleagues recommended to always push to cache, to get a better populated cache.